VisualStudio.gitignore

```diff
@@ -174,8 +174,10 @@ rcf/
 # Windows Store app package directories and files
 AppPackages/
 BundleArtifacts/
-Package.StoreAssociation.xml
 _pkginfo.txt
+_scale-*.appx
+_language-*.appx 
```

Comment:
Whitespace is important with these rules. @bondarenkod please have a look at this.